common/components/componentsBySection/Donate/DonateBlurb.jsx

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import React from 'react';
+
+type State = {|
+  blurbType: ?string
+|};
+
+class DonateBlurb extends React.Component<{||}, State> {
+  constructor(): void {
+    super();
+    this.blurbs = {
+      "GivingTuesday": this._givingTuesday
+    };
+    this.state = {
+      blurbType: window.DONATE_PAGE_BLURB
+    };
+  }
+  
+  render(): React$Node {
+    const showBlurb: boolean = this.state.blurbType in this.blurbs;
+    return (
+      <React.Fragment>
+        {showBlurb && this.blurbs[this.state.blurbType]()}
+      </React.Fragment>
+    );
+  }
+  
+  _givingTuesday(): React$Node {
+    return (
+      <React.Fragment>
+        <h1>
+          Giving Tuesday 2020 is on Dec. 1st
+        </h1>
```

Comment:
```suggestion
        <h2>
          Giving Tuesday 2020 is on Dec. 1st
        </h2>
```

In the short term this makes the title text a little smaller but also has the H1/H2/H3 ... hierarchy.